ioctl/cmd/bc/bucket.go

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) 2020 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package bc
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/grpc-ecosystem/go-grpc-middleware/util/metautils"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc/status"
+
+	"github.com/iotexproject/iotex-proto/golang/iotexapi"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+)
+
+// Multi-language support
+var (
+	bcBucketCmdShorts = map[config.Language]string{
+		config.English: "Get bucket for given index on IoTeX blockchain",
+		config.Chinese: "在IoTeX区块链上根据索引读取投票",
+	}
+	bcBucketUses = map[config.Language]string{
+		config.English: "bucket [BUCKET_INDEX]",
+		config.Chinese: "bucket [票索引]",
+	}
+)
+
+// bcBucketCmd represents the bc Bucket command
+var bcBucketCmd = &cobra.Command{
+	Use:   config.TranslateInLang(bcBucketUses, config.UILanguage),
+	Short: config.TranslateInLang(bcBucketCmdShorts, config.UILanguage),
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		err := getBucket(args[0])
```

Comment:
we'll need to parse out a list of index from args[i]
like 47 56 77 86 100 372 7328 72727

Reply:
i am not going to change ioctl for now. it is a client side changes which is easier if we need to this. API is harder to change later on, so it is better to change it now